eventbus/jetstream/sensor/trigger_conn.go

```diff
@@ -257,15 +257,25 @@ func (conn *JetstreamTriggerConn) processMsg(
 		conn.Logger.Errorf("can't get Metadata() for message %+v??", m)
 	}
 
-	defer func() {
-		err = m.AckSync() // this call does the acknowledgment and waits for an acknowledgment from the server (that it received the ack) too
-		if err != nil {
-			errStr := fmt.Sprintf("Error performing AckSync() on message: %v", err)
-			conn.Logger.Error(errStr)
-		}
-
-		conn.Logger.Debugf("acked message of Stream seq: %s:%d, Consumer seq: %s:%d", meta.Stream, meta.Sequence.Stream, meta.Consumer, meta.Sequence.Consumer)
+	ch := make(chan bool)
+	go func () {
+	  ticker := time.NewTicker(500 * time.Millisecond)
+	  for {
+	    select {
+	    case <- ch:
+	      err = m.AckSync()
```

Comment:
We should re-add the logging, and probably also log in the where `m.InProgress()` returns an error

Reply:
Agreed. I'll do that.